tests/integration/targets/dcnm_interface/tests/dcnm/dcnm_lo_fabric.yaml

```diff
@@ -0,0 +1,199 @@
+          - '(result["diff"][0]["deploy"] | length) == 0'
+
+    - assert:
+        that:
+          - 'item["RETURN_CODE"] == 200'  
+      loop: '{{ result.response }}'
+
+    - name: Query and save new data for loopback0
+      cisco.dcnm.dcnm_interface:
+        check_deploy: True
+        fabric: "{{ ansible_it_fabric }}" 
+        state: query
+        config:
+          - name: lo0
+            switch:
+              - "{{ ansible_switch2 }}"
+      register: lpk0_data_new
+
+    - assert:
+        that:
+          - 'lpk0_data_new.response[0]["interfaces"][0]["nvPairs"]["IP"] == "192.168.5.1"'
+
+  # Only execute this test if loopback0 and 1 are fabric loopbacks
+  when:
+    - lpk0_data.response[0]['policy'] == 'int_fabric_loopback_11_1'
+    - lpk1_data.response[0]['policy'] == 'int_fabric_loopback_11_1'
+
+##############################################
+##             CLEANUP                      ##
+##############################################
+
+  always:
+
+  - name: Set Loopback0 and Loopback1 back to inital state
+    cisco.dcnm.dcnm_interface:
+      check_deploy: True
+      fabric: "{{ ansible_it_fabric }}" 
+      state: merged
+      config:
+        - name: lo0
+          type: lo
+          switch:
+            - "{{ ansible_switch2 }}"
+          deploy: true
+          profile:
+            admin_state: true
+            mode: fabric
+            ipv4_addr: "{{ lpk0_data.response[0]['interfaces'][0]['nvPairs']['IP'] }}"
+            # secondary_ipv4_addr: "{{ lpk0_data.response[0]['interfaces'][0]['nvPairs']['SECONDARY_IP'] }}"
+            ipv6_addr: "{{ lpk0_data.response[0]['interfaces'][0]['nvPairs']['V6IP'] }}"
+            route_tag: "{{ lpk0_data.response[0]['interfaces'][0]['nvPairs']['ROUTE_MAP_TAG'] }}"
+            description: "{{ lpk0_data.response[0]['interfaces'][0]['nvPairs']['DESC'] }}"
+
+        - name: lo1
+          type: lo
+          switch:
+            - "{{ ansible_switch2 }}"
+          deploy: true
+          profile:
+            admin_state: true
+            mode: fabric
+            ipv4_addr: "{{ lpk1_data.response[0]['interfaces'][0]['nvPairs']['IP'] }}"
+            # secondary_ipv4_addr: "{{ lpk1_data.response[0]['interfaces'][0]['nvPairs']['SECONDARY_IP'] }}"
+            ipv6_addr: "{{ lpk1_data.response[0]['interfaces'][0]['nvPairs']['V6IP'] }}"
+            route_tag: "{{ lpk1_data.response[0]['interfaces'][0]['nvPairs']['ROUTE_MAP_TAG'] }}"
+            description: "{{ lpk1_data.response[0]['interfaces'][0]['nvPairs']['DESC'] }}"
+
+  - name: Put fabric to default state
+    cisco.dcnm.dcnm_interface:
+      check_deploy: True
+      fabric: "{{ ansible_it_fabric }}" 
+      state: overridden                     # only choose form [merged, replaced, deleted, overridden, query]
+    register: result
+    when: IT_CONTEXT is not defined
+
+  - assert:
+      that:
+        - 'item["RETURN_CODE"] == 200'  
+    loop: '{{ result.response }}'
```

Comment:
I assume that there will be atleast one fabric loopback interface. Otherwise the tests may not run

Reply:
Yes, the test will only run if Loopback0 and Loopback1 are both fabric interfaces.  I think this is a safe assumption given that we are controlling the test environment.

Reply:
I just made a new update to the integration tests to be more complete